docs/user-guide/api-mediation/configuration-jwt.md

```diff
@@ -35,16 +35,63 @@ For more information, see [Configure Passtickets](configuration-extender-passtic
 1. Open the file `zowe.yaml`.
 2. Configure the following properties:
 
-    * **components.gateway.apiml.security.allowtokenrefresh: true**  
-    Add this property to enable the refresh endpoint.
+  * **components.gateway.apiml.security.allowtokenrefresh: true**  
+  Add this property to enable the refresh endpoint.
 
-    * **components.gateway.apiml.security.zosmf.applid**  
-    If you use z/OSMF as an authentication provider, provide a valid `APPLID`. The API ML generates a passticket for the specified `APPLID` and subsequently uses this passticket to authenticate to z/OSMF. The default value in the installation of z/OSMF is `IZUDFLT`.
+  * **components.gateway.apiml.security.zosmf.applid**  
+  If you use z/OSMF as an authentication provider, provide a valid `APPLID`. The API ML generates a PassTicket for the specified `APPLID` and subsequently uses this PassTicket to authenticate to z/OSMF. The default value in the installation of z/OSMF is `IZUDFLT`.
 
 3. Restart Zowe.
 
 ## Authorization
 
-Authorization is a method used to determine access rights of an entity.
+Authorization is used to set the access rights of an entity.
+
+In the API ML, authorization is performed by any of the following z/OS security managers:
+* [ACF2](https://www.broadcom.com/products/mainframe/identity-access/acf2)
+* [IBM RACF](https://www.ibm.com/support/knowledgecenter/zosbasics/com.ibm.zos.zsecurity/zsecc_042.htm)
+*  [Top Secret](https://www.broadcom.com/products/mainframe/identity-access/top-secret). 
+
+An authentication token is used as proof of valid authentication. The authorization checks, however, are always performed by the z/OS security manager.
+
+## Additional customizable properties when using JWT tokens
+
+You can also customize the following properties when authenticating with a JWT token:
+
+* **components.gateway.apiml.security.auth.zosmf.ServiceId**  
+    This parameter specifies the z/OSMF service id used as authentication provider. The service id is defined in the static definition of z/OSMF. The default value is `zosmf`.
+
+* **apiml.security.auth.tokenProperties.expirationInSeconds**  
```

Comment:
This should be components.gateway.apiml.security.auth.tokenProperties.expirationInSeconds